doc/filters/invoke.rst

```diff
@@ -0,0 +1,19 @@
+``invoke``
+==========
+
+The ``invoke`` filter invokes an arrow function with the given arguments:
+
+.. code-block:: twig
+
+    {% set person = { first: "Bob", last: "Smith" } %}
+    {% set func = p => "#{p.first} #{p.last}" %}
+
+    {{ func|invoke(person) }}
+    {# outputs Bob Smith #}
+
+Note that the arrow function has access to the current context.
```

Comment:
I don't understand this sentence. Do you mean that the function does **not** have access to the context?

Reply:
I took this sentence is from the documentation of `map`. Maybe in this case we would have to be more specific, that the arrow function has access to the context of where it is _defined_? Or would you prefer to remove it alltogether?

Reply:
It's true for `map` because we explicitely pass the context to the anonymous function that Twig creates. But here, this is not the case. This sentence should be removed.